modern/src/reports/ReportLayoutPage.js

```diff
@@ -29,6 +34,18 @@ const ReportLayoutPage = ({ reportFilterForm:ReportFilterForm, setItems, childre
             <Paper className={classes.form}>
               <ReportFilterForm setItems={ setItems } />
             </Paper>
+            {showChartType && (
```

Comment:
I just noticed one big thing here. I don't think it makes sense to include chart here. It's used only in the chart report.

Reply:
What would you like to suggest where it should be shifted?

Should I move it again to the right side as before?

Reply:
Maybe we need to split layout differently somehow to allow it?

Reply:
Hi Anton,

I am unable to understand how to implement this? Can you give me a direction?

One solution I thought that includes the Report Filer component and Chart component in a single ChartReportPage.

Thanks

Reply:
Maybe you can pass filter using props. Something like this:
```js
        filter={
          <ReportFilterForm setItems={ setItems } />
          <Paper className={classes.chart}>
            <FormControl variant="filled" margin="normal" fullWidth>
            ...
        }
```

Reply:
Done. Please check and suggest improvements.